lib/oxidized/node.rb

```diff
@@ -155,11 +155,13 @@ def resolve_input opt
     end
 
     def resolve_output opt
-      output = (opt[:output] or Oxidized.config.output.default)
-      if not Oxidized.mgr.output[output]
-        Oxidized.mgr.add_output output or raise MethodNotFound, "#{output} not found for node #{ip}"
+      outputs = (opt[:output] or Oxidized.config.output.default)
```

Comment:
Would be nice to have consistent naming. You've renamed `output` to `outputs` in some places but left it as it is in another places.